### ISSUE TITLE
Change plugins manager buttons from disabled/enabled to enable/disable (Trac #92)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#96]
Migrated from http://microfluidics.utoronto.ca/microdrop/ticket/92
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:43", 
    "description": "", 
    "reporter": "ryan", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763583719301", 
    "component": "microdrop", 
    "summary": "Change plugins manager buttons from disabled/enabled to enable/disable", 
    "priority": "minor", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-04-03T15:05:48", 
    "milestone": "Microdrop 1.0", 
    "owner": "", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120403T15:10:59: 
* ryan@fobel.net commented:

> commit b9008edb2c1a3d1a749c1529f02672d45c735c3d
> Author: Ryan Fobel <ryan@fobel.net>
> Date:   Tue Apr 3 11:07:00 2012 -0400
> 
>     Change plugin disabled/enabled to enable/disable
>     
>     Fixes #96.


* ryan@fobel.net changed resolution from "" to "fixed"
* ryan@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:43: ryan changed component from "" to "microdrop"
